Documentation/ceph-object.md

```diff
@@ -57,7 +67,59 @@ kubectl create -f object.yaml
 kubectl -n rook-ceph get pod -l app=rook-ceph-rgw
 ```
 
+### Connect to an External Object Store
+
+Rook can connect to existing RGW gateways, this works in conjunction with the external mode of the `CephCluster` CRD.
+So if you have an external `CephCluster` CR, you can instruct Rook to consume external gateways with the following:
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: my-store
+  namespace: rook-ceph
+spec:
+  gateway:
+    # the port on which **ALL** the gateway(s) are listening on
```

Comment:
How about not including comments here? If they want details we can link to the ceph-object-store-crd.yaml later in this section.